app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PostStoreRequest;
+use App\Http\Requests\PostUpdateRequest;
+use App\Services\postService;
+use Illuminate\Http\Request;
+use App\Traits\APIResponse;
+
+class PostController extends ControllerWithGuard
+{
+    use APIResponse;
+
+    private $postService;
+
+    public function __construct(postService $postService)
+    {
+        parent::__construct();
+        $this->postService = $postService;
+    }
+
+    public function index()
+    {
+        //
+    }
+
+    public function store(PostStoreRequest $request)
+    {
+        $data = [
+            'user_id'=> $request->user()->id,
```

Comment:
Please use const from model
```suggestion
            Post::USER_ID => $request->user()->id,
```